docs/source/guide/get_started/installation.rst

```diff
@@ -79,71 +79,23 @@ Install OpenVINO™ Training Extensions for users (XPU devices)
 *************************************************************
 
 1. Install OpenVINO™ Training Extensions
-package:
-
-* A local source in development mode
+from source to use XPU functionality.
 
 .. tab-set::
 
-    .. tab-item:: PyPI
-
-        .. code-block:: shell
-
-            pip install otx
-
-    .. tab-item:: Source
+    .. tab-item:: Minimum requirements
 
         .. code-block:: shell
 
             # Clone the training_extensions repository with the following command:
             git clone https://github.com/openvinotoolkit/training_extensions.git
             cd training_extensions
 
-            # Set up a virtual environment.
-            python -m venv .otx
-            source .otx/bin/activate
-
-            pip install -e .
-
-2. Install Intel Extensions for Pytorch & Requirements
-for training according to your system environment.
-
-.. tab-set::
-
-    .. tab-item:: Minimum requirements
-
-        .. code-block:: shell
-
-            pip install '.[xpu]' --extra-index-url https://pytorch-extension.intel.com/release-whl/stable/xpu/us/
+            pip install -e '.[xpu]' --extra-index-url https://download.pytorch.org/whl/test/xpu
 
-        .. note::
+[Optional] Refer to the `PyTorch documentation guide <https://pytorch.org/docs/stable/notes/get_start_xpu.html>`_
```

Comment:
I think it's worth mentioning that one may need to visit the official pytorch page to install prerequisites. Now, there is an impression that this link is an optional information provided just in case

Reply:
Added https://github.com/openvinotoolkit/training_extensions/pull/4048/commits/cc1103dd8894f317e0991835990267130734dcaa